MTM-Frontend/src/pages/Auth/Register.tsx

```diff
@@ -33,6 +35,12 @@ interface FormValues {
   affiliation?: string;
 }
 
+interface organization {
+  id: number;
```

Comment:
Let's Refactor this into the types folder for api call

Reply:
In that case I am going to move the FormValues to the FormTypes and the organization type to a new folder called ApiCallTypes